server/public/fix.css

```diff
@@ -0,0 +1,34 @@
+body,
+.c,
+h6,
+h5,
+h4,
+h3,
+h2,
+h1,
+.btn {
+  font-family: "Raleway", "HelveticaNeue", "Helvetica Neue", Helvetica, Arial, sans-serif;
+}
+
+p,
+h6,
+h5,
+h4,
+h3,
+h2,
+h1 {
+  margin-block-start: 0;
+}
+
+.btn {
+  font-size: 1em;
+  font-weight: bold;
+  text-transform: none;
+  letter-spacing: normal;
+}
+
+@media(min-width:35em) {
+  .row {
+      border-spacing: 10px 0;
```

Comment:
As lit is using `em` units for defining any dimension, it's probably better to stick to this pattern when overwriting values in there.